launchmany.py

```diff
@@ -47,7 +47,7 @@ def main(tiles_list, path, start_year):
             if walltime > 48:
                 print('Capping walltime at maximum 48 hrs, was', walltime)
                 walltime = 48
-
+            # note // should we change the log output directory?
```

Comment:
Yep - we should at least make this configurable, defaulting to the current directory.  In another pull though - you may want to create another branch 😉